prediction_market_agent_tooling/tools/cow/cow_order.py

```
@@ -4,6 +4,29 @@
+    OrderQuoteSide1,
+    OrderQuoteSideKindSell,
+    OrderStatus,
+    TokenAmount,
+)
+from eth_account.signers.local import LocalAccount
+from eth_typing.evm import ChecksumAddress
+from web3 import Web3
+from web3.types import Wei
+
+from prediction_market_agent_tooling.config import APIKeys
+from prediction_market_agent_tooling.gtypes import ChecksumAddress, Wei, wei_type
```

Ruff (0.8.2)

Redefinition of unused `ChecksumAddress` from line 21

(F811)